modeltranslation/tests/tests.py

```diff
@@ -108,6 +117,20 @@ def tearDown(self):
         super().tearDown()
         trans_real.activate(self._old_language)
 
+    def assert_db_record(self, instance, **expected_fields):
+        """
+        Compares field values stored in the db.
+        """
+        with connection.cursor() as cursor :
+            cursor.execute(
+                f"select {', '.join(expected_fields)} from {instance._meta.db_table} where id = {instance.pk}",
+            )
+            row = dictfetchall(cursor)
+            self.assertEqual(
+                row[0],
+                expected_fields,
+            )
```

Comment:
```suggestion
        actual = type(instance).objects.filter(pk=instance.pk).values(*expected_fields.keys())[0]
        assert actual == expected_fields
```

Maybe something like this is better?

And i think it's better to write it as separate function with two args,  `assert_db_record(instance, {"title": "...", ...})`,

Reply:
Your option doesn't extract exact values from the database. Modeltranslation rewrites a query to fetch a value of the default translation field (`"title_de"`) for an original field(`"title"`)

Reply:
I mean `Model.objects.values("title")` performs `select title_de from table`

Reply:
Oh, yes, Use `.rewrite(False)` like other tests here.